### PR TITLE
Custom diffs and confusion table

### PR DESCRIPTION
**Pull request by DanielSpecht**

### CODE REVIEW

File: evaluate-confusion.lisp [hunk 1]
```diff
@@ -0,0 +1,125 @@
+(in-package :cl-conllu)
+
+(defun parent-upostag (token sent)
+  (if (=(token-head token) 0)
+      "root"
+      (token-upostag (token-parent token sent))))
+
+
+(defun token-to-parent-distance(token sent)
```
Comment (arademaker) [inline evaluate-confusion.lisp:10]: space before `(`

Comment (arademaker) [inline evaluate-confusion.lisp:10]: add space before `(`

File: packages.lisp [hunk 2]
```diff
@@ -31,6 +31,13 @@
 	   #:write-conllu-to-stream   
 	   #:levenshtein
 	   #:diff
+
+	   #:diff-parent-upostag
```
Comment (arademaker) [inline packages.lisp]: do we need to export auxiliary functions?

Comment (DanielSpecht) [reply packages.lisp]: The functions implemented are specific ways to perfor a diff in the sentences and building and saving the resulting confusion table. The library already exports a function which aims to make a diff of sentences (cl-conllu:diff). In my interpretation, exporting the functions adds to this type of 'diff' features. But I understand that in the moment they solve very specific needs of the team and may not add to the library as a generic tool for manipulating conllu files.

**Pull request merged**